app/controllers/api/v3/users/games/categories/pbs_controller.rb

```diff
@@ -32,7 +32,7 @@ def set_category
   end
 
   def set_pb
-    unless @pb = @user.pb_for(@category)
+    unless (@pb = @user.pb_for(@category))
```

Comment:
Let's actually ditch this whole `[if/unless] thing = other_thing` idea, this is too easy to misread as `[if/unless] thing == other thing`, or to read it correctly and think it was a typo.

Reply:
I personally like when things are on separate lines as well since it reduces confusion for myself because I assume that it's a boolean expression.